src/test/tasks.yaml

```
@@ -49,5 +49,15 @@ tasks:
             address: demo.uds.dev/status/202
             code: 202
 
+      - cmd: uds zarf tools kubectl get pods -n podinfo -o=jsonpath='{.items[0].metadata.name}'
+        setVariables:
+          - name: PODINFO_NAME
+      - description: Check for podinfo to come back up
+        wait:
+          cluster:
+            kind: Pod
+            name: ${PODINFO_NAME}
+            namespace: podinfo
```

```suggestion
      - description: Check for podinfo to come back up
        wait:
          cluster:
            kind: Pod
            name: app.kubernetes.io/name=podinfo
            namespace: podinfo
```
nit: this supports labels so we should be able to skip the "get pod name" part.